package.json

```diff
@@ -19,11 +20,19 @@
     "test:ci": "node --expose-gc --trace-warnings --unhandled-rejections=strict ./node_modules/jest/bin/jest.js --bail --ci --colors --expand --errorOnDeprecated --coverage --reporters=default --maxWorkers=1 --forceExit --logHeapUsage",
     "test:manual": "jest --expand --colors --watch",
     "test:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --expand --colors --watch --runInBand",
-    "package": "yarn build && node scripts/package.js"
+    "prepublishOnly": "yarn build"
```

Comment:
is it possible to leave previous commands names? 

cause `build` now is not running building - it's running packing

Reply:
in some sence we 'build' the lib)